_config.yml

```
@@ -3,6 +3,7 @@ title: Pyronear
 description: "Détection précoce de feux de forêt" # this means to ignore newlines until "baseurl:"
 baseurl: "https://pyronear.github.io/" # the subpath of your site, e.g. /blog/
 url: "" # the base hostname & protocol for your site
+tags: [pyronear,association,deeplearning,feux,incendies,raspberry,dataforgood,atraksis]
```

```suggestion
tags: [pyronear,association,deeplearning,feux,incendies,détection,forêt,raspberry,dataforgood,atraksis]
```

Even if these words are in description, shouldn't we add "forêt" and "détection" in keywords tags ?